app/assets/javascripts/components/revisions/diff_viewer.jsx

```diff
@@ -116,13 +115,19 @@ const DiffViewer = React.createClass({
         dataType: 'jsonp',
         url: diffUrl,
         success: (data) => {
-          const revisionData = data.query.pages[this.props.revision.mw_page_id].revisions;
+          const firstRevisionData = data.query.pages[this.props.revision.mw_page_id]
+                                      .revisions[0];
+          let lastRevisionData = null;
+          if (data.query.pages[this.props.revision.mw_page_id].revisions.length === 2) {
```

Comment:
I would assign this condition to a variable with a meaningful name for readability. ;)

i.e.
```javascript
hasRevisions = 'Something hard to read or to understand by humans'
if(hasRevisions)
  doSomething 
```